### PR TITLE
feat: hidden

### PR DESCRIPTION
**Pull request by HeleenSG**
- Added hidden helper class and documentation

### CODE REVIEW

File: manon/visually-hidden.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    clip-path: polygon(0px 0px, 0px 0px, 0px 0px, 0px 0px);
+    white-space: nowrap;
+    overflow: hidden;
+    width: 0;
+    height: 0;
+    max-width: 0;
+    max-height: 0;
+
+    /* Overwriting any specifics that might have been set.    */
+    /* !important is used here to solve any specificity issues */
```
Comment (TwistMeister) [inline manon/visually-hidden.scss:14]: Thanks for the thorough comment. This is imo just what !important should be used for, but a little explanation on this is great for future reference or other devs that will come across this.

File: docs/documentation/components/hidden.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+          <li><a href="#variables">Instelbare variabelen</a></li>
+        </ul>
+      </nav>
+
+      <article>
+        <div>
+          <section id="introduction">
+            <h1>Verbergen - <dfn>hidden</dfn></h1>
+            <p>Object verbergen. Door middel van een <code>display:none</code> wordt het object verborgen.</p>
+            <p><strong>Let op:</strong> Screen readers verbergen over het algemeen ook objecten met een <code>display:none</code> voor hun gebruikers. Als dit niet de bedoeling is en de content wel beschikbaar dient te zijn voor screenreader gebruikers is de helper class <a href="">screenreader-only</a> waarschijnlijk de betere optie.</p>
```
[the remark this answers is not in the document]
Comment (TwistMeister) [reply docs/documentation/components/hidden.html:56]: Also, shouldn't the <a href="">screenreader-only</a> point to the docs on that particular class? So the reader can look up more information about that specific subject?

[the remark this answers is not in the document]
Comment (HeleenSG) [reply docs/documentation/components/hidden.html:56]: Agreed, I will update this. Thank you!

[the remark this answers is not in the document]
Comment (TwistMeister) [reply docs/documentation/components/hidden.html]: The hyperlink "screenreader-only" still doesn't link to anything. Is this intentional?

**Review by TwistMeister — APPROVED** [review]

**Review by ppvg — APPROVED** [review]

**Pull request merged**